sfepy/base/compat.py

```diff
@@ -8,10 +8,14 @@
 __all__ = ['in1d', 'unique']
 
 try:
-    in1d = nm.in1d
+    in1d = nm.isin
 
 except AttributeError:
-    in1d = nm.setmember1d
+    try:
+        in1d = nm.in1d
+
+    except AttributeError:
+        in1d = nm.setmember1d
 
 unique = nm.unique
```

Comment:
Import of `nm.unique1d` could also be removed. Couldn't it?

Reply:
Ir is imported only in `compat.py`, so that the code still works with older numpy versions.

Reply:
But it seems unused, so yes, it can be removed.